src/ansible_navigator/actions/__init__.py

```diff
@@ -1,4 +1,17 @@
-"""actions for Explorer"""
+"""A package containing all available actions.
+
+The actions package is a plugin like implementation for all available application actions.
+
+Actions can be added without registration and are discovered and loaded
+when their ``KEGEX`` is matched against user input.
```

Comment:
(minor, maybe for follow-ups)
I wonder if this `KEGEX` thing should be marked with a `:term:` role and maybe belongs to a Sphinx glossary.

Reply:
for the sake of history, KEGEX was a mash-up of key and regex from the early days of ansible-navigator when it was only single-key driven.....  It probably should be renamed REGEX, which more accurately describes it, or maybe ACTION_REGEX.....  it would not be a trivial change, but a good experience for someone not familiar with how it is used

Reply:
:+1: